homeassistant/components/statistics/sensor.py

```diff
@@ -100,10 +115,21 @@ def __init__(self, entity_id, name, sampling_size, max_age, precision):
         self.min_age = self.max_age = None
         self.change = self.average_change = self.change_rate = None
         self._update_listener = None
+        self._debounced_updater = debounced_updater
 
     async def async_added_to_hass(self):
         """Register callbacks."""
 
+        async def _async_debounced_update():
```

Comment:
Why only set this in `async_added_to_hass` ? You can do this in the constructor.

Reply:
I was playing with `self.hass._` at start (which I only have access in added, I think), but yes, I'll move it. 

Another doubt I had was where to instance the Debouncer, in setup or in entity init. It's ok in setup, right?